### PR TITLE
Feat: use import/order and sort-keys rules

### PR DESCRIPTION
**Pull request by rodrigo4244**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -37,6 +37,9 @@
     "guard-for-in": "off",
     "import/no-dynamic-require": "off",
     "import/no-extraneous-dependencies": ["error", { "devDependencies": ["**/test/**"] }],
+    "import/order": [ "error", { 
+      "groups": [ "builtin", "external", "internal", [ "parent", "sibling" ], "index" ]
```
Comment (rodrigo4244) [inline .eslintrc.json]: This does not add a new line though, just separate by `builtin`, `external`, `internal` and then `parent`, `sibling` (these two does not matter the order) and `index` (Local index -> `'./ '`)

Comment (rodrigo4244) [reply .eslintrc.json]: I didn't put the `newline` feature because we cannot merge groups :(

**Comment by rodrigo4244** [comment]
These rules are going to break Emissary, Osmosis, Postman and Gear. But we can fix this on next PRs.

**Comment by keenahn** [comment]
I will leave it to you to submit PR's to fix the styles that are now violating the rules :)

**Pull request merged**